bindings/java/src/org/sleuthkit/datamodel/OsAccountRealmManager.java

```diff
@@ -337,11 +346,12 @@ private OsRealmUpdateResult updateRealm(OsAccountRealm realm, String realmAddr,
 			String currRealmName = realmNames.isEmpty() ? null : realmNames.get(0);	// currently there is only one name.
 
 			// Update realm name if:
-			//	 Current realm name is empty
-			//	 The passed in realm name is not empty
-			//	 The address is not a special windows address
-			if (StringUtils.isBlank(currRealmName) && StringUtils.isNotBlank(realmName) && 
-					((currRealmAddr == null || !currRealmAddr.equals(WindowsAccountUtils.SPECIAL_WINDOWS_REALM_ADDR)))) {
+			//  current SID (if exists)  is a wellknown windows SID, and the passed in realm name is not empty and is not the same as current realm name 
```

Comment:
Put space in "wellknown", and make sure indent is consistent with the line below